### DISCUSSION TITLE
RSS post description taking from the wrong place

### DISCUSSION THREAD

**Discussion by 0xtiago** [gethinode/hinode#1317] [Help]
Hi,
I've been facing a small issue when trying to share my content through RSS. Hinode generates the feed automatically, but there is a issue on it, instead of filling the `<description>` XML tag with the `description: ` tag of .md metadata, it's taking the description from the first paragraph of the blog post.

Bellow the example using the Hinode Demo SIte:
- https://demo.gethinode.com/en/blog/image/
- https://demo.gethinode.com/en/blog/index.xml

![image](https://github.com/user-attachments/assets/054981ed-6868-4e49-96bc-757a8d10ee4e)

Did somebody face the same problem and fixed it?

Thank you!

Tiago

**Comment by d-oit** [comment]
Hugo use this for rss

https://gohugo.io/content-management/summaries/

https://github.com/gohugoio/hugo/blob/master/tpl%2Ftplimpl%2Fembedded%2Ftemplates%2F_default%2Frss.xml

Only replace summary with description or
 use chatgpt, Claude like this


---

Prompt:

Create a custom RSS layout template for Hugo by modifying the default template from the Hugo GitHub repository. Replace the summary field with the description field for each item. Ensure the new template is complete, functional, and ready for deployment. Reference: Hugo Default RSS Template. https://github.com/gohugoio/hugo/blob/master/tpl%2Ftplimpl%2Fembedded%2Ftemplates%2F_default%2Frss.xml


---

**Reply by markdumay** [reply]
Thanks for the suggestion @d-oit!

**Comment by markdumay** [comment]
Currently, Hinode does not include a custom RSS template. As @d-oit suggested, you can [define a custom RSS template](https://gohugo.io/templates/rss/#custom-templates) quite easily. I'd be happy to include it in the Hinode repository if it fixes any issues. Likewise, do let me know if there's a bug in the Hinode codebase.
